.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: java
+jdk:
+  - oraclejdk8
```

Comment:
selenium 3.X requires java8, which is not available to chose in "node_js" language setting in travis